shared/src/packages/meditation/libs/validation-schemas/create-meditation-entry-form.validation-schema.ts

```diff
@@ -0,0 +1,29 @@
+import joi from 'joi';
+
+import { ContentType } from '#libs/enums/enums.js';
+import { FileUploadValidationRule } from '#packages/files/files.js';
+
+import { MeditationEntryValidationMessage } from '../enums/enums.js';
+import { type MeditationEntryCreateForm } from '../types/types.js';
+
+const createMeditationEntryForm = joi.object<MeditationEntryCreateForm, true>({
+  name: joi.string().trim().required().messages({
+    'any.required': MeditationEntryValidationMessage.NAME_REQUIRED,
+    'string.empty': MeditationEntryValidationMessage.NAME_REQUIRED,
+  }),
+  file: joi.object().required().unknown(true).messages({
+    'object.base': MeditationEntryValidationMessage.FILE_REQUIRED,
+  }),
+  fileType: joi.string().valid(ContentType.MPEG).required().messages({
+    'any.only': MeditationEntryValidationMessage.MPEG_REQUIRED,
+  }),
+  fileSize: joi
+    .number()
+    .max(FileUploadValidationRule.MAXIMUM_FILE_SIZE)
+    .required()
+    .messages({
+      'number.max': MeditationEntryValidationMessage.SIZE_TOO_BIG,
+    }),
```

Comment:
It's useless

Reply:
Already changed as with te comment above